README.md

```diff
@@ -16,7 +16,6 @@
   - [:hammer\_and\_wrench: Built With ](#hammer_and_wrench-built-with-)
     - [Tech Stack ](#tech-stack-)
     - [Key Features ](#key-features-)
-  - [🦻 WalkThrough Video ](#walkthrough-demo-)
   - [💻 Getting Started ](#-getting-started-)
     - [Prerequisites](#prerequisites)
     - [Setup](#setup)
```

Comment:
- [ ] Kindly fix the following broken links in the table of contents:

- The about link should link to the about part of your project, kindly fix it and rename it to reflect this project
![image](https://github.com/joyapisi/blog_app/assets/81932478/c114bfc6-8336-47b6-b0cf-fcea7867c7a1)


- Ensure the user is taken to the Author section when they click on the Author section link
![image](https://github.com/joyapisi/blog_app/assets/81932478/0cab8f70-637a-44fa-81d1-2154e9c2e510)